docs/Components/AttachedComponent.md

```diff
@@ -59,4 +59,51 @@ print(math.degrees(aileron.angle))
 plane.angle = math.radians(5)
 print(math.degrees(aileron.angle))
 >>> 14.99999999999
+```
+##Angle:
+The property angle will return to you the angle that the given component is set.
+
+Let's say we have a component 'Wing' attached to the body of the plane, that it's set as a FreeBody Component called 'Plane'. The property angle will take in count the plane angle and the relative angle of the wing, just as shown in the example below:
+
+``` python
+env = Ambient()
+plane = FreeBody(
+    name='plane',
+    type='vehicle',
+    mass=2.0,
+    position_cg=Vector2(x=-0.05, y=0),
+    pitch_rot_inertia=30.0,
+    ambient=env
+)
+
+wing = AttachedComponent(
+    name='wing',
+    type='wing',
+    mass=0.3,
+    relative_position=Vector2(x=-0.15, y=0.2)
+)
+
+plane.angle = math.radians(3.2)
+wing.relative_angle = math.radians(1)
+print(math.degrees(wing.angle))
+>>> 4.2
```

Comment:
Despite being possible to set the "relative_angle" property after the component instantiation, we are setting as a convention to always set it on instantiation, together with relative_position.

In the future we are going to make this mandatory, so the user always set it and the library don't face calculation problems.